README.md

```diff
@@ -92,6 +92,27 @@ We'd love for you to contribute to our open source projects. Before we can accep
 - If you **have a feature request**, open an issue or submit an implementation via a pull request
 - If you **want to contribute**, submit a pull request.
 
+### Development
+
```

Comment:
Please move this to contributing.md

Reply:
Contributing is information to show before filing an issue or PR, which is after local development is started. Most things like this are put into a README section or wiki page